uefi-raw/src/protocol/disk.rs

```diff
@@ -64,3 +64,42 @@ impl DiskIo2Protocol {
     pub const GUID: Guid = guid!("151c8eae-7f2c-472c-9e54-9828194f6a88");
     pub const REVISION: u64 = 0x00020000;
 }
+
+pub const DISK_INFO_IDE_INTERFACE_GUID: Guid = guid!("5e948fe3-26d3-42b5-af17-610287188dec");
+pub const DISK_INFO_UFS_INTERFACE_GUID: Guid = guid!("4b3029cc-6b98-47fb-bc96-76dcb80441f0");
+pub const DISK_INFO_USB_INTERFACE_GUID: Guid = guid!("cb871572-c11a-47b5-b492-675eafa77727");
+pub const DISK_INFO_AHCI_INTERFACE_GUID: Guid = guid!("9e498932-4abc-45af-a34d-0247787be7c6");
+pub const DISK_INFO_NVME_INTERFACE_GUID: Guid = guid!("3ab14680-5d3f-4a4d-bcdc-cc380018c7f7");
+pub const DISK_INFO_SCSI_INTERFACE_GUID: Guid = guid!("08f74baa-ea36-41d9-9521-21a70f8780bc");
+pub const DISK_INFO_SD_MMC_INTERFACE_GUID: Guid = guid!("8deec992-d39c-4a5c-ab6b-986e14242b9d");
```

Comment:
Let's move these to associated constants of `DiskInfoProtocol` (the `DISK_INFO_` prefix can be dropped then).